assets/widgets/checkmarkbutton.json

```diff
@@ -36,11 +36,13 @@
             "data": {
               "texture": "joinbutton",
               "scale": 0.8,
-              "anchor": [0.5,0.5]
+              "anchor": [0.5, 0.5]
             },
             "layout": {
               "x_anchor": "center",
-              "y_anchor": "middle"
+              "y_anchor": "middle",
+              "x_offset": 0.0,
```

Comment:
Don't think you need to add the offset here?

Reply:
Actually i took a look at the code again. The placeholder offset fields were needed to get rid of error messages about missing fields for widgets.